src/uu/mktemp/src/mktemp.rs

```diff
@@ -441,7 +442,7 @@ pub fn uu_app() -> Command {
         .arg(Arg::new(ARG_TEMPLATE).num_args(..=1))
 }
 
-pub fn dry_exec(tmpdir: &str, prefix: &str, rand: usize, suffix: &str) -> UResult<()> {
+pub fn dry_exec(tmpdir: &str, prefix: &str, rand: usize, suffix: &str) -> UResult<PathBuf> {
```

Comment:
Do you happen to know why this is `pub`?

Reply:
Nope. It did seem a little odd.